ow_dynamic_terrain/scripts/modify_terrain_grinder_pub.py

```diff
@@ -21,15 +21,17 @@ class ModifyTerrainGrinder:
   def __init__(self, *args):
     rospy.init_node("modify_terrain_grinder_pub", anonymous=True)
     self.last_translation = np.zeros(3)
-    self.pub = rospy.Publisher(
-        'ow_dynamic_terrain/modify_terrain_circle', modify_terrain_circle, queue_size=1)
+    self.pub_visual = rospy.Publisher(
+        'ow_dynamic_terrain/modify_terrain_circle/visual', modify_terrain_circle, queue_size=1)
+    self.pub_collision = rospy.Publisher(
+        'ow_dynamic_terrain/modify_terrain_circle/collision', modify_terrain_circle, queue_size=1)
     self.states_sub = rospy.Subscriber(
         "/gazebo/link_states", LinkStates, self.handle_link_states)
 
-  def compose_modify_terrain_circle_message(self, position):
+  def compose_modify_terrain_circle_message(self, position, scale=1.0):
     return modify_terrain_circle(position=Point(position.x, position.y, position.z),
-                                 outer_radius=0.008, inner_radius=0.001,
-                                 weight=-0.2, merge_method="min")
+                                 outer_radius=0.008*scale, inner_radius=0.001*scale,
```

Comment:
Where do the numbers 0.008 and 0.001 come from? Please add a comment to explain the logic here. If there is to be a default size for a circle, I would expect it to be a unit circle.

Reply:
addressed